specs/zkevm-prover.md

```diff
@@ -23,16 +23,16 @@
 
 ## Overview
 
-Prover produces a so called [ZK fault proof][g-zk-fault-proof] that states a [state][g-state] transition from `S` to
-`S'` is valid. It sounds like there's no big differences from validity proof. That's true. But the point is this is used
+The Prover produces a [ZK fault proof][g-zk-fault-proof] that states that a [state][g-state] transition from `S` to
```

Comment:
Since Prover can be any one of several people, wouldn't it be more appropriate to use 'A' instead of 'The'?

```diff
-The Prover
+A Prover
```

Reply:
Good catch!